### DISCUSSION TITLE
Add DNS records for Firebase email verification custom domain

### DISCUSSION THREAD

**Discussion by craig1123** [vercel/vercel#4169] [General] [answered]
I bought a domain from Vercel and I use Firebase for authentication and email verification. In order for the email to be from my domain, Firebase needs to me to verify it by adding 4 DNS records to my domain registrar. 

I got the TXT records uploaded by doing  ```now dns add <domain> @ TXT 'v=spf1 include:_spf.firebasemail.com ~all'```

When I try adding the CNAME by doing ```now dns add firebase1._domainkey.<domain> @ CNAME mail-<domain>.<firebasestuff>.firebasemail.com.```
I get `The provided domain name "firebase1._domainkey.<domain>" is invalid`. How do add the firebase CNAME?

<img width="694" alt="Screen Shot 2020-04-27 at 1 35 31 PM" src="https://user-images.githubusercontent.com/17581359/80419122-64bb8a80-888d-11ea-8f64-f8714f85db56.png">

**Comment by coetry** [comment, marked as answer]
Hi! Can you try the following pattern:

`now dns add <domain> <host> CNAME <value>`

That should work for you. We're working on making editing DNS from the dashboard a possibility soon since using the CLI for this can be tricky!

**Reply by craig1123** [reply]
It worked! Thank you

**Reply by NickCarducci** [reply]
Sorry to bother, but I am getting `invalid number of arguments` from `now dns add carducci.sh '@' TXT 'v=spf1 include:spf.migadu.com -all'`

I also tried escaping spaces in the `value` with `\` and surrounding it instead with `"``"` to no avail.  

I'll just use the dashboard under my domain to add the rest of the `admin migadu` `dns` records.  That worked.  I'll send this note anyway.  Thanks!
